regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -5,6 +5,8 @@ This contains all of the styles specific to the TOC
 */
 .regulation-nav {
 
+    padding-top: 70px;
```

Comment:
Doesn't this depend on the length of the text? For example, here's the CFR with this change:
<img width="269" alt="screen shot 2016-04-13 at 6 16 59 pm" src="https://cloud.githubusercontent.com/assets/326918/14511211/f2e50f90-01a3-11e6-901d-3220166ce9c2.png">

And here's CFR changes:
<img width="415" alt="screen shot 2016-04-13 at 6 17 24 pm" src="https://cloud.githubusercontent.com/assets/326918/14511232/0dfd5e36-01a4-11e6-804a-9d4f558070b0.png">

One potential solution: if you've got a fixed header size (say, exactly 70px), you can have a fixed amount of padding.